### ISSUE TITLE
Vagrant box not booting with latest hypernode_php7 (build 3436)

### ISSUE THREAD

**Issue by paales** [EcomDev/fast-hypernode#3]
Running OSX

I have the following config:

```
name 'domainname'
hostname name + '.box'
domains %w(www.domainname.box)
profiler false
developer true
magento2 true
install true
shell true
php7 true
cpu 2
memory 2048
varnish false
```

Getting the following output (When starting the box with 2992 it works as expected):

```
Bringing machine 'hypernode' up with 'virtualbox' provider...
==> hypernode: AutoNetwork assigning "33.33.33.3" to 'hypernode'
==> hypernode: Importing base box 'hypernode_php7'...
==> hypernode: Matching MAC address for NAT networking...
==> hypernode: Checking if box 'hypernode_php7' is up to date...
==> hypernode: Setting the name of the VM: fast-hypernode_hypernode_1477849118642_50637
==> hypernode: Clearing any previously set network interfaces...
==> hypernode: Preparing network interfaces based on configuration...
    hypernode: Adapter 1: nat
    hypernode: Adapter 2: hostonly
==> hypernode: Forwarding ports...
    hypernode: 22 (guest) => 2222 (host) (adapter 1)
==> hypernode: Running 'pre-boot' VM customizations...
==> hypernode: Booting VM...
==> hypernode: Waiting for machine to boot. This may take a few minutes...
    hypernode: SSH address: 127.0.0.1:2222
    hypernode: SSH username: vagrant
    hypernode: SSH auth method: private key
    hypernode:
    hypernode: Vagrant insecure key detected. Vagrant will automatically replace
    hypernode: this with a newly generated keypair for better security.
    hypernode:
    hypernode: Inserting generated public key within guest...
    hypernode: Removing insecure key from the guest if it's present...
    hypernode: Key inserted! Disconnecting and reconnecting using new SSH key...
==> hypernode: Machine booted and ready!
[hypernode] GuestAdditions versions on your host (5.1.8) and guest (5.0.26) do not match.
stdin: is not a tty
Reading package lists...
Building dependency tree...
Reading state information...
dkms is already the newest version.
linux-headers-3.13.0-100-generic is already the newest version.
linux-headers-3.13.0-100-generic set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
Copy iso file /Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso into the box /tmp/VBoxGuestAdditions.iso
stdin: is not a tty
mount: block device /tmp/VBoxGuestAdditions.iso is write-protected, mounting read-only
Installing Virtualbox Guest Additions 5.1.8 - guest version is 5.0.26
stdin: is not a tty
Verifying archive integrity... All good.
Uncompressing VirtualBox 5.1.8 Guest Additions for Linux...........
VirtualBox Guest Additions installer
Removing installed version 5.0.26 of VirtualBox Guest Additions...
Stopping VirtualBox Additions ...fail!
(Cannot unload module vboxguest)
Removing existing VirtualBox DKMS kernel modules ...done.
Removing existing VirtualBox non-DKMS kernel modules ...done.
update-initramfs: Generating /boot/initrd.img-/3.13.0-100-generic
update-initramfs: Generating /boot/initrd.img-/3.2.0-113-generic
update-initramfs: Generating /boot/initrd.img-/3.2.0-23-generic
Copying additional installer modules ...
Installing additional modules ...
vboxadd.sh: Building Guest Additions kernel modules.
vboxadd.sh: You should restart your guest to make sure the new modules are actually used.
vboxadd.sh: Starting the VirtualBox Guest Additions.

Could not find the X.Org or XFree86 Window System, skipping.
stdin: is not a tty

Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   5.0.26
VBoxService inside the vm claims: 5.1.8
Going on, assuming VBoxService is correct...
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   5.0.26
VBoxService inside the vm claims: 5.1.8
Going on, assuming VBoxService is correct...
==> hypernode: Checking for guest additions in VM...
    hypernode: The guest additions on this VM do not match the installed version of
    hypernode: VirtualBox! In most cases this is fine, but in rare cases it can
    hypernode: prevent things such as shared folders from working properly. If you see
    hypernode: shared folder errors, please make sure the guest additions within the
    hypernode: virtual machine match the version of VirtualBox you have installed on
    hypernode: your host and reload your VM.
    hypernode:
    hypernode: Guest Additions Version: 5.0.26
    hypernode: VirtualBox Version: 5.1
==> hypernode: Setting hostname...
==> hypernode: Configuring and enabling network interfaces...
==> hypernode: Installing nfs server on the guest...
==> hypernode: Exporting NFS shared folders from guest...
==> hypernode: Preparing to edit /etc/exports on the guest...
==> hypernode: Mounting NFS shared folders from guest...
    hypernode: /data/web => server
==> hypernode: Updating /etc/hosts file on active guest machines...
==> hypernode: Updating /etc/hosts file on host machine (password may be required)...
==> hypernode: Running provisioner: shell...
    hypernode: Running: /var/folders/4l/y0r50fwd30l1zv25nkp3mcy80000gp/T/vagrant-shell20161030-66473-gm4r3l.sh
==> hypernode: stdin: is not a tty
==> hypernode: solr            memory          limited
==> hypernode: Running provisioner: shell...
    hypernode: Running: /var/folders/4l/y0r50fwd30l1zv25nkp3mcy80000gp/T/vagrant-shell20161030-66473-kq4kj8.sh
==> hypernode: stdin: is not a tty
==> hypernode: Running provisioner: shell...
    hypernode: Running: /var/folders/4l/y0r50fwd30l1zv25nkp3mcy80000gp/T/vagrant-shell20161030-66473-9f0223.sh
==> hypernode: stdin: is not a tty
==> hypernode: Changing user id
==> hypernode: nginx stop/waiting
==> hypernode: php7.0-fpm stop/waiting
==> hypernode: stop: Unknown instance:
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by IvanChepurnyi** [comment]
Seems like in new version of box they removed hhvm for PHP7 boxes.

**Comment by IvanChepurnyi** [comment]
Fixed in 1.2.4

**Comment by paales** [comment]
Thanks!
